docs/quickstart.md

```diff
@@ -69,6 +69,17 @@ window.onresize = () => {
 };
 ```
 
+## Coordinate System
+
+The Gizmo examples demonstrate the Y-up coordinate system, which aligns with the Three.js default coordinate system.
+
+The default directions for the Y-up coordinate system are as follows:
+- Up: `Y+`
+- Right: `X+`
+- Forward: `Z-`
+
+If you need to use a different coordinate system, refer to the [Z-up](./examples/z-up) and [X-up](./examples/x-up) sections.
+
```

Comment:
Some additional info about the supported coordinate systems and quick links to others.